templates/form-data.json

```diff
@@ -15,7 +15,7 @@
         {
           "title": "Tell us about your project",
           "id": "about-your-project",
-          "noCommentsFromLead": true,
+          "noCommentsFromLead": false,
```

Comment:
Is there a reason for this change?

Reply:
`noCommentsFromLead`, when true means that we will not include this field in comments.
There is no field on marketo for "tell us about your project", hence if not part of comments it will not reach to the marketo.
You may check the fields on marketo using this link:
(replace the form-id with relevant form-id)
https://experience.adobe.com/#/@canonical/so:066-EOV-335/marketo-engage/mktform/<form-id>/overview/details